interpreter/cling/lib/MetaProcessor/MetaSema.cpp

```diff
@@ -372,9 +372,17 @@ namespace cling {
                              "\t\t\t\t  'decl' dump ast declarations\n"
                              "\t\t\t\t  'undo' show undo stack\n"
       "\n"
-      "   " << metaString << "help\t\t\t- Shows this information\n"
+      "   " << metaString << "R [user@]host[:dir] [-l user] [-d dbg] [script]\t- Launch process in a remote host\n"
       "\n"
-      "   " << metaString << "q\t\t\t\t- Exit the program\n"
+      "   " << metaString << "demo\t- Launch GUI demo\n"
+      "\n"
+      "   " << metaString << "credits\t- Show credits\n"
+      "\n"
+      "   " << metaString << "license\t- Show license\n"
+      "\n"
+      "   " << metaString << "help\t\t\t- Shows this information (also .?)\n"
+      "\n"
+      "   " << metaString << "q\t\t\t\t- Exit the program (also .quit or .exit)\n"
```

Comment:
All these are ROOT-specific.  Standalone cling does not know about `.R`, etc..

This should be moved to the "ROOT special commands" section in `TApplication.cxx`.

Reply:
Changed.

Reply:
Moved this stuff to https://github.com/root-project/root/issues/10065